src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1536,7 +1536,12 @@ The file `PROACTIVE_HOME/config/security.java.policy-server` allows to configure
 
 * Deploy ProActive Nodes
 * Execute jobs
-* Pause the Scheduler
+* Perform administrator actions from the scheduler portal like:
+** Start the scheduler: Jobs can be submitted. Getting the jobs results is possible.
+** Stop the scheduler: Jobs cannot be submitted anymore. This terminates every submitted jobs. Getting the jobs results is possible.
+** Freeze the scheduler: Every running tasks are terminated, but the pending jobs wait for the scheduler to resume.
+** Pause the scheduler: Every running jobs are terminated.
+** Kill the scheduler: Nothing can be done.
```

Comment:
After we kill scheduler. Is it possible to access already finished jobs?

Reply:
For the kill, you should precise what happen to the jobs and tasks.

Reply:
@Mykhailenko  : I will add "Similar to Ctrl-C. No interation can be done anymore."

Reply:
@jrochas but i think only saying  "Similar to Ctrl-C. No interation can be done anymore."   can be enough

Reply:
Question, is the scheduler process automatically terminated when we click on kill ?

If not, it would make sense to do this, as it would not be equivalent to Ctrl-C otherwise.